src/unicode_range.h

```diff
@@ -0,0 +1,6 @@
+#ifndef YOUVE_GOTTA_LOVE_CLANG
+#define YOUVE_GOTTA_LOVE_CLANG
```

Comment:
Not many people will know the history behind this joke.

Reply:
I also wouldn't blame clang too hard. For example, I had this logic in yescrypt for a while, actually expecting non-gcc compilers to fail otherwise:

```c
#if defined(__GNUC__) && !defined(__clang__) && !defined(__ICC)
#pragma GCC push_options
#pragma GCC optimize ("unroll-loops")
#endif
```